sys-devel/gcc/gcc-5.4.0_2016_07_08.recipe

```diff
@@ -0,0 +1,347 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```

Comment:
HTTPS is not only about avoiding tampering with the source, but also hiding what specific file you wanted. HTTP sends plain requests, including the vhost and filename. Not something most of us really care about right now, but in theory some people might, which is the reason for projects like [HTTPS-Everywhere](https://www.eff.org/https-everywhere).